OOVPADatabase/D3D8_OOVPA.inl

```diff
@@ -307,6 +307,10 @@ OOVPATable D3D8_OOVPA[] = {
     REGISTER_OOVPAS(D3DDevice_SetPixelShader, 3911, 4034, 4627),
     REGISTER_OOVPAS(D3DDevice_SetPixelShaderConstant, 3911, 4831),
     REGISTER_OOVPAS(D3DDevice_SetPixelShaderProgram, 3911),
+    REGISTER_OOVPAS(D3DDevice_SetRenderState_Simple, 3911), // Final generic OOVPA: 3911; Removed: 0
+    REGISTER_OOVPAS(D3DDevice_SetRenderStateNotInline, 3911), // Final generic OOVPA: 3911; Removed: 0 // NOTE: Must be after D3DDevice_SetRenderState_Simple.
```

Comment:
This reliance on specific OOVPA registration ordering, is a deviation of the otherwise sorted registration approach.
What would happen if ordering of this OOVPA was alphanumerical?
Wouldn't the regular repeated searches produce the same result?

Reply:
Correct, though we do want to do our best effort to have one time scan than repeated scan (excluding manual scan).

Eventually later on in TestOOPVAs function may will add report of certain signatures need to be re-arrange to reduce repeat scans to hopefully just one scan (excluding manual scan).